src/diet.ts

```diff
@@ -307,6 +307,8 @@ function menu(): MenuItem<Note>[] {
     new MenuItem($item`extra-greasy slider`),
     new MenuItem(mallMin(lasagnas)),
     new MenuItem(mallMin(smallEpics)),
+    new MenuItem($item`Miserable Pie`),
```

Comment:
These items need to be in the potionMenu or it won't consider their extra value from the buffs